components/app.js

```diff
@@ -0,0 +1,26 @@
+class App {
+    constructor(gradeTable){
+        this.gradeTable = gradeTable;
+        this.handleGetGradesError = this.handleGetGradesError.bind(this);
+        this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+    }
+    handleGetGradesError(error){
+        console.error('this is the error: ',[error]);
+    }

```

Comment:
Since this is only going to display one error, it's unnecessary to wrap it in an `Array`.